packages/middleware-ssec/src/index.ts

```diff
@@ -78,3 +77,16 @@ export const getSsecPlugin = (config: PreviouslyResolved): Pluggable<any, any> =
     clientStack.add(ssecMiddleware(config), ssecMiddlewareOptions);
   },
 });
+
+function isValidBase64EncodedSSECustomerKey(str: string, options: PreviouslyResolved) {
```

Comment:
please add two unit or integration tests, one with a passing example and one with a failing example